spark/v3.1/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -117,16 +131,26 @@ public static void stopMetastoreAndSpark() throws Exception {
     spark = null;
   }
 
+  @After
+  @Override
+  public void cleanup() throws IOException {
+    super.cleanup();
+    dropTable("test3");
+  }
+
   @Override
   protected Table createTable(String name, Schema schema, PartitionSpec spec) {
     Table table = catalog.createTable(TableIdentifier.of("default", name), schema);
     TableOperations ops = ((BaseTable) table).operations();
     TableMetadata meta = ops.current();
     ops.commit(meta, meta.upgradeToFormatVersion(2));
+    table
+        .updateProperties()
+        .set(TableProperties.PARQUET_VECTORIZATION_ENABLED, String.valueOf(vectorized))
+        .commit();
```

Comment:
`TableProperties.PARQUET_VECTORIZATION_ENABLED_DEFAULT` changed from false to true since this code was written. Thus, before, we only set `TableProperties.PARQUET_VECTORIZATION_ENABLED` explicitly if vectorized is true, but since `TableProperties.PARQUET_VECTORIZATION_ENABLED_DEFAULT` changed to true, all the tests in this class have been running only with vectorized reads and not with non-vectorized reads at all!
This fixes that issue.